onyo/commands/tree.py

```diff
@@ -9,27 +9,37 @@
 log = logging.getLogger('onyo')
 
 
-def sanitize_directories(directories: list, opdir: str) -> list[str]:
+def sanitize_directories(repo: Repo, directories: list[str]) -> list[str]:
     """
     Check a list of directories. If any do not exist or are a file, and error
     will be printed.
 
     Returns a string of the valid directories relative to opdir.
     """
     dirs = []
-    error_path = []
+    path_not_in_repo_error = []
+    no_directory_error = []
```

Comment:
Sorry to nit. Please feel free to say if I'm descending into bike shedding.

I just see that these two vars basically do the same thing, yet the naming convention is completely different, which then (incorrectly) implies that they are quite far apart in utility.

Perhaps something similar to:
```
error_path_not_in_repo = []
error_path_not_dir = []
```

Reply:
I'm comfortable with the variables being separate. IMO, that's a matter of choice, regarding how specific the error messages should be. I have no problems with the direction you chose here.

+1 on the renaming for consistency. Thanks. :-)